processor/src/test/java/org/mapstruct/ap/test/subclassmapping/SubclassMappingTest.java

```diff
@@ -66,19 +81,38 @@ void existingMappingsAreUsedWhenFound() {
     }
 
     @ProcessorTest
-    void subclassMappingInheritsMapping() {
-        VehicleCollection vehicles = new VehicleCollection();
-        Car car = new Car();
-        car.setVehicleManufacturingCompany( "BenZ" );
-        vehicles.getVehicles().add( car );
+    @WithClasses( SimpleSubclassMapper.class )
+    void subclassMappingInheritsInverseMapping() {
+        VehicleCollectionDto vehiclesDto = new VehicleCollectionDto();
+        CarDto carDto = new CarDto();
+        carDto.setMaker( "BenZ" );
+        vehiclesDto.getVehicles().add( carDto );
 
-        VehicleCollectionDto result = SimpleSubclassMapper.INSTANCE.map( vehicles );
+        VehicleCollection result = SimpleSubclassMapper.INSTANCE.mapInverse( vehiclesDto );
 
         assertThat( result.getVehicles() )
-            .extracting( VehicleDto::getMaker )
+            .extracting( Vehicle::getVehicleManufacturingCompany )
```

Comment:
Don't we care only about the return type? I thought that this is what we are testing? At least that is what we test in the original method

Reply:
That's is already covered in the previous test (line 57). This one is that the `@Mapping` is also inherited by the inverse subclass mapping.

Reply:
You are right, my mistake